packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -419,12 +419,18 @@ function filterModeToAdditiveKarmaModifier(mode: FilterMode): number {
 }
 
 function filterModeToMultiplicativeKarmaModifier(mode: FilterMode): number {
-  if (typeof mode === "number" && 0 < mode && mode < 1) {
+  // Example: "x10.0" is a multiplier of 10
+  const match = typeof mode === "string" && mode.match(/^x(\d+(?:\.\d+)?)$/);
```

Comment:
cc @b0b3rt for awareness, in the long run I would actually like to get rid of the rule where between 0 and 1 means it's multiplicative